### PR TITLE
Adding fully styled example article

### PR DESCRIPTION
**Pull request by narphorium**
Here's a sample that could serve as a starting point for publishers looking to create their own custom AMPHTML template.

### CODE REVIEW

**Comment by wulab** [comment]
@narphorium You have many of unclosed h3 tags.

**Comment by dvoytenko** [comment]
@narphorium @erwinmombay `layout=responsive` is only supported for `<amp-carousel type=slides>`. This is because we were not able to support height-based aspect-ratio for items in the default `<amp-carousel>`. But we will keep trying. For now the default carousel supports `fixed-height` layout, e.g.  `<amp-carousel layout layout=fixed-height height=X>`. For this layout you just need to specify `height` and it will take all of the available width.

tl;dr
There are two modes in `amp-carousel`: slides (`type=slides`) and default carousel. They do similar things by giving horizontal layout for multiple items, but they actually work very differently. Slides show only one item at a time and thus each item is sized to fit the container. Default carousel shows several items at a time, however many can fit horizontally, much like a filmstrip. The issue we ran into is that with Slides, since each item is sized by container, it's very easy to support width-based `layout=responsive` because the width is known beforehand and the height is easily calculated in CSS by applying the aspect ration. This requires zero javascript and works very fast w/o causing reflows. Carousel is harder. We can't size content by container. What we'd really like to do in this case is to size the items' height by the container and calculate width using aspect ratio in CSS (in other words do exactly the opposite thing we do for Slides). However, we haven't found a way to do so in CSS yet.

**Comment by erwinmombay** [comment]
@dvoytenko thanks for the thorough explanation! i'll pull this down and tweak around with it so we can get this merged :)

**Comment by rsimha-amp** [comment]
No idea how this got closed. All I did was run `git fetch --prune origin` on my machine. Huh!
